client/src/pages/SignAll.js

```diff
@@ -6,6 +6,17 @@ import Link from "@material-ui/core/Link";
 import Typography from "@material-ui/core/Typography";
 import { makeStyles } from "@material-ui/core/styles";
 import Container from "@material-ui/core/Container";
+import Snackbar from "@material-ui/core/Snackbar";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import ErrorIcon from "@material-ui/icons/Error";
+import InfoIcon from "@material-ui/icons/Info";
+import WarningIcon from "@material-ui/icons/Warning";
+import CloseIcon from "@material-ui/icons/Close";
+import { amber, green } from "@material-ui/core/colors";
+import SnackbarContent from "@material-ui/core/SnackbarContent";
+import clsx from "clsx";
+import IconButton from "@material-ui/core/IconButton";
+
 import { useLocation } from "react-router";
```

Comment:
React router missing from package.json dependencies

Reply:
update to react-router-dom, i believe react-router is a dependency of react-router-dom or react so there is no need to npm install it again.